client/my-sites/plans/jetpack-plans/product-store/hooks/use-products-to-display.ts

```diff
@@ -13,8 +13,7 @@ export const useProductsToDisplay = ( { siteId, duration }: ProductsListProps )
 		useGetPlansGridProducts( siteId );
 
 	const allAvailableProducts = useMemo( () => {
-		// TODO - need to add Jetpack social product constants and data
-		const moreAvailableProducts = [ ...JETPACK_BOOST_PRODUCTS, ...JETPACK_CRM_PRODUCTS ].map(
```

Comment:
This is the wrong place to be adding JETPACK_BOOST_PRODUCTS into the product grid.  All non-external products should be added from the `useGetPlansGridProducts()` hook so that we can check first if the product is already owned for that site. So I went ahead and fixed it here, and added both Boost and Social from within the `useGetPlansGridProducts()` hook.  (See the `client/my-sites/plans/jetpack-plans/use-get-plans-grid-products.ts` file.)